rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/util/locations/Locations.java

```diff
@@ -122,7 +122,7 @@ public static ISourceLocation toLoc(String uri) {
         try {
             return URIUtil.createFromURI(uri);
         } catch (UnsupportedOperationException e) {
-            if (e.getMessage().contains("Opaque URI schemes are not supported")) {
+            if (e.getMessage() != null && e.getMessage().contains("Opaque URI schemes are not supported")) {
```

Comment:
Why is this code here, is it not more logical to do this check in `createFromURI`?